Project/Assets/Prairie/Framework/Script/Interaction/Annotation.cs

```diff
@@ -49,6 +49,8 @@ public class Annotation : Interaction
     private readonly float BOX_WIDTH = Screen.width / 2;
     private readonly float BOX_HEIGHT = Screen.height - 20;
 
+	private FirstPersonInteractor player;
+
```

Comment:
This variable shouldn't be stored as a property on `Annotation`. The two places we read to it, we reassigned it 2 lines back (within scope).

I would remove this property, and replace references to it with local variables.